packages/shared/blockchain/evm-based/src/lib/evm-based.utils.ts

```diff
@@ -86,6 +89,30 @@ export const evmBasedUtils = {
     return signedTransaction.rawTransaction
   },
 
+  validateSenderBalance: async (client: Web3, privateKey: string, tx: TransactionConfig) => {
```

Comment:
this works for One and Klaytn and XDC? All 3 has different client libs then web3

Reply:
It is verified from the e2e tests to work for the chains